### DISCUSSION TITLE
[Question] Why the database on heroku doesn't work?

### DISCUSSION THREAD

**Discussion by adrianbeys** [piekny27/PulseBook#17] [Q&A] [answered]
Why the database on heroku doesn't work?

Is it related to the database type? 🤔

**Comment by OlexTix** [comment]
I recommend checking the connection to the database (such as credentials, database location/address). Another good thing to check would be to verify database's integrity.
This also can be related to user's permissions.

**Comment by piekny27** [comment, marked as answer]
Heroku free plan is slug server type, so you cant modify files on deployed application. The way to bypass this is change your local database to uri database. e.g. you can use postgresql uri database delivered by heroku, but you have to add it in addons at heroku dashboard page.
